h5d_public.go

```diff
@@ -0,0 +1,16 @@
+// Copyright ©2019 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package hdf5
+
+// #include "hdf5.h"
+import "C"
+
+const (
+	// Used to unset chunk cache configuration parameter.
+	// https://support.hdfgroup.org/HDF5/doc/RM/RM_H5P.html#Property-SetChunkCache
+	D_CHUNK_CACHE_NSLOTS_DEFAULT int     = -1 // rdcc_nslots
```

Comment:
we could perhaps add a somewhat better comment, piggy-backing on the official documentation. (especially for `rdcc_w0`.)
(at the very least, we should explain what `rdcc` stands for.)

Reply:
Fixed